app/components/web-components/vl-document/component.js

```diff
@@ -60,44 +80,9 @@ export default Component.extend(UploadDocumentMixin, {
       this.set('isVerifyingDelete', true);
     },
 
-    async toggleConfidential(document) {
-      document.toggleProperty('confidential');
-      await document.save();
-    },
-
-    chooseAccessLevel(accessLevel) {
-      this.set('preferredAccessLevel', accessLevel);
-    },
-
-    async saveChanges() {
-      let preferredAccessLevel = this.get('preferredAccessLevel');
-      this.toggleProperty('isEditingAccessLevel');
-      let documentVersion = this.get('documentVersion');
-      if (preferredAccessLevel) {
-        await documentVersion.set('accessLevel', preferredAccessLevel);
-        await documentVersion.save();
-      }
-      this.resetPreferredAccessLevel();
```

Comment:
Hier verwijder je blokken code die gebruik maken van preferredAccessLevel.
Maar preferredAccessLevel wordt nergens anders meer gebruikt dus mag dat ook allemaal weg.

Reply:
Zou in orde moeten zijn.